packages/orbit-components/src/Modal/index.tsx

```diff
@@ -403,19 +422,18 @@ const Modal = React.forwardRef<Instance, Props>(
           !isMobileFullPage && "bg-[black]/50",
           "lm:overflow-y-auto lm:p-1000 lm:bg-[black]/50",
         )}
-        // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
-        tabIndex={0}
+        tabIndex={-1}
         onKeyDown={handleKeyDown}
         onScroll={handleScroll}
         onClick={handleClickOutside}
         data-test={dataTest}
         id={id}
         ref={modalBodyRef}
         role="dialog"
-        // eslint-disable-next-line jsx-a11y/no-autofocus
-        autoFocus={autoFocus}
         aria-modal="true"
-        aria-labelledby={hasModalTitle ? modalTitleID : ""}
+        aria-label={ariaLabel}
+        aria-labelledby={ariaLabelledby || (hasModalTitle ? modalTitleID : "")}
+        aria-describedby={ariaDescribedby || (hasModalDescription ? modalDescriptionID : "")}
```

Comment:
I would replace empty string with `undefined` here, as currently without title/description it ends up with invalid values. Visible in With Itinerary story.